index.js

```diff
@@ -65,6 +65,7 @@ var parseOptions = module.exports._parseOptions = function (options) {
   options.compile = options.compile != null ? options.compile : true;
   options.compress = options.compress != null ? options.compress : isProduction;
   options.gzip = options.gzip != null ? options.gzip : false;
+  options.fingerprints = options.fingerprints || isProduction ? true : false
```

Comment:
This isn’t actually correct — I’ll make the change when merging though. With this line, you cannot disable fingerprints in production. If you follow the pattern of the lines above, you’re able to override the value regardless of environment.